appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/properties/YoungAndroidComponentSelectorPropertyEditor.java

```diff
@@ -24,12 +24,12 @@
  *
  * @author lizlooney@google.com (Liz Looney)
  */
-public final class YoungAndroidComponentSelectorPropertyEditor
+public class YoungAndroidComponentSelectorPropertyEditor
```

Comment:
Create some form of abstraction (extra layer?) to make use of this functionality in the Data Source selection case